about.html

```diff
@@ -11,54 +11,112 @@
 
 </head>
 
-<body>
+<body>  
     <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        <nav>
+            <img class="logo" src="./img/lambda-black.png">
+            <a href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="">Products</a>
```

Comment:
make sure you at least put a # in the href='' section or it will break the page!